### ISSUE TITLE
Make application accessible

### ISSUE THREAD

**Issue by peholmst** [vaadin/whereabouts#4]
Accessibility has not been a priority while building the first version of this application. This should be addressed so that the application is as accessible as it can be, given what's possible with Vaadin today.
